### PR TITLE
QA rotation automation and ticket template updates

### PR DESCRIPTION
**Pull request by TKDickson**
## Description of Change

- Swapping out me for Tom on all the things, since I'm rotating into/he's rotating out of floater.
- Added myself to the nightly detox build failure tickets, as I'm more likely to take at least some of them in the floater position if I can beat RB online some days.
- Removed the broken RC testing graph that I've deleted from the RC ticket both of the last 2 sprints

## Testing

- [ ] Tested on iOS 
- [ ] Tested on Android 

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [ ] PR is connected to issue(s)
- [ ] Tests are included to cover this change (when possible)
- [ ] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [ ] No secrets or API keys are checked in
- [ ] All imports are absolute (no relative imports)
- [ ] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Post_RC_Code_Change.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Post RC Code Change
 about: Template for Adding Code After RC is Cut requiring a new RC build
 title: Post RC Code Change Template
 labels: release
-assignees: djultratom, bischoffa
+assignees: TKDickson, bischoffa
```
Comment (timwright12) [inline .github/ISSUE_TEMPLATE/Post_RC_Code_Change.md:6]: @TKDickson adam should be removed

File: .github/workflows/release_branch_issue.yml [hunk 2]
```diff
@@ -110,7 +110,7 @@ jobs:
       - name: Notify Slack for release coordination
         env:
           SLACK_API_TOKEN: ${{ secrets.SLACK_API_TOKEN }}
-          SLACK_DJUltraTom: $DJUltraTom
+          SLACK_TKDicksom: $TKDickson
```
Comment (timwright12) [inline .github/workflows/release_branch_issue.yml:113]:
```suggestion
          SLACK_TKDickson: $TKDickson
```

**Review by timwright12 — CHANGES_REQUESTED** [review]
Just a couple comments

File: .github/ISSUE_TEMPLATE/Post_RC_Code_Change.md (hunk 1 above)
Comment (TKDickson) [reply .github/ISSUE_TEMPLATE/Post_RC_Code_Change.md:6]: @timwright12 I thought about it, but I'd rather wait a few days to get Sarah's github info, and then be able to find and replace all of the Adam references with Sarah's username

Comment (timwright12) [reply .github/ISSUE_TEMPLATE/Post_RC_Code_Change.md:6]: Can you drop a ticket in here for that so we can close the loop?

Comment (TKDickson) [reply .github/ISSUE_TEMPLATE/Post_RC_Code_Change.md:6]: Sure, #9828

Comment (IsraelleHub) [reply .github/ISSUE_TEMPLATE/Post_RC_Code_Change.md:6]: looks good!

**Review by timwright12 — APPROVED** [review]

**Pull request merged**